cmd/unity/cmd/changeResolver.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 The CUE Authors
```

Comment:
nit: Go filenames are generally snake_case.go, not camelCase.go

Reply:
Done. Also fixed other filenames in this PR (straight rename so figured that would be fine to bundle)